docs/learn-staking.md

```diff
@@ -502,6 +502,18 @@ strain of the network due to peer-to-peer message passing. The estimate of the n
 that Polkadot will have at maturity is around 1000. Kusama, Polkadot's canary network, currently has
 900 validator slots in the active set.
 
+## Motion #103: New Minimum Nomination Bond
+
+At the time of writing (#6,339,031), Polkadot has a maximum nominator count of 22,500, and the current staking configuration prevents new nominators from joing. Motion #103 proposes new nomination limits to the Polkadot network, offering a temporary solution to increasing the stability and security of the network. Note that this motion **does not** increase the maximum nominator count.
```

Comment:
-  " ...offering a temporary solution to increasing the stability and security of the network."
  - increasing -> increase